.github/workflows/push.yml

```diff
@@ -359,7 +359,7 @@ jobs:
           gpgcheck=1
           gpgkey=https://cli-dist.keboola.com/rpm/keboola.gpg
           " | tee /etc/yum.repos.d/keboola.repo
-          dnf install -y kbc
+          dnf install -y keboola-cli
```

Comment:
Instalacia balickov po release sa testuje vramci release CI pipeline, takze ak je niekde preklep, tak na to prideme jednoducho.